src/machine.py

```diff
@@ -27,19 +27,16 @@ def __init__(self, status, machineId, buildingId, machineType, timeLeft, isAvail
 		Check if all the attributes are valid and assigns them if they are
 		Raises ValueError if attributes are invalid
 		"""
-		allowedStatuses = ["success", "failure"]
-		if status not in allowedStatuses:
-			raise ValueError(
-					"Invalid value for 'status' ({0}), must be one of {1}"
-					.format(status, allowedStatuses))
+		if status is None:
+			raise ValueError("Invalid value for 'status', must not be 'None'")
 		
 		if machineId is None:
 			raise ValueError("Invalid value for 'machineId', must not be 'None'")
 			
 		if buildingId is None:
 			raise ValueError("Invalid value for 'buildingId', must not be 'None'")
 			
-		allowedMachineTypes = ["washer", "dryer"]
+		allowedMachineTypes = ["Washer", "Dryer"]
```

Comment:
Might be fine to leave this as-is and do `if machineType.lower() not in allowedMachineTypes`. But it's fine with me either way.